infrastructure/assumptions/tools/ahem-generate-table.py

```diff
@@ -30,52 +30,66 @@ def grouper(n, iterable):
 per_row = 17
 
 
-def build_header(is_test):
-    rv = []
-
-    rv.append("""
-<!doctype html>
-<title>Ahem checker</title>""")
-
-    if is_test:
-        rv.append("""
-<link rel="match" href="ahem-ref.html">""")
-
-    rv.append("""
-<style>""")
-
-    if not is_test:
-        rv.append("""
-@font-face {
-  font-family: Ahem;
-  src: url("../../fonts/Ahem.ttf");
-}""")
-
-    rv.append("""
+doctype = "<!doctype html>"
+title = "<title>Ahem checker</title>"
+style_open = """
+<style>
 * {
   padding: 0;
   margin: 0;
   border: none;
 }
-
+td {
+  width: 34px;
+}""".strip()
+style_close = "</style>"
+style_font_face = """
+@font-face {
+  font-family: Ahem;
+  src: url("../../fonts/Ahem.ttf");
+}""".strip()
+style_table_with_font = """
 table {
   font: 15px/1 Ahem;
   border-collapse: separate;
   border-spacing: 1px;
   table-layout: fixed;
-}
+}""".strip()
+style_table_no_font = """
+table {
+  font: 15px/1;
```

Comment:
This declaration is invalid; we should probably just have `font-size: 15px; line-height: 1;` in this case.